### DISCUSSION TITLE
How to add "custom locations"?

### DISCUSSION THREAD

**Discussion by smooth-baking-powder** [tobychui/zoraxy#329] [Q&A]
Hi,

at the moment i test Zoraxy and i didnt find an option to add custom location with other rules.

for example where can i deny internet access of /wp-admin in wordpress? In other Reverse Proxies i can use custom locations and deny internet access.

or for /synapse-admin
allow ip/24;
deny all;

Thanks

**Comment by tobychui** [comment]
@corincorvus Currently Zoraxy do not support subpath access rule controls. But you can do that with a single `.htaccess` file under the `wp-admin` directory.

**Reply by smooth-baking-powder** [reply]
Thats not an Option for other Server, so i cant use Zoraxy at the moment

**Reply by smooth-baking-powder** [reply]
Hey, Any updates here? :)

**Reply by tobychui** [reply]
> Hey, Any updates here? :)

I am still waiting for your pull request :)

**Reply by smooth-baking-powder** [reply]
i am not able to programming :/ 
Iam good for testing, administrating, configure things but programming i never did.

**Reply by schwankd** [reply]
Locations are a must for me too, im also not able to do smth in code but would be greatfull for that feature

**Reply by Shekelme** [reply]
I agree with the previous comment. This is a very necessary feature.

**Reply by anubis-genix** [reply]
That's a feature I'd love to see in Zoraxy. There's already an opened issue about this: https://github.com/tobychui/zoraxy/issues/238

**Reply by tobychui** [reply]
Thanks for bring this up again. I think currently the plugin api could be used to do this exact feature. 

Additionally, I think the recent AI coding agent should be intelligent enough to implement this for you (since all the api are already in the doc). You can vibe code a Zoraxy plugin doing exactly that with Codex gpt4.5 High or Claude Opus 4.8 High and upload to the Zoraxy App Store if you want.
